Assets/MRTK/Core/Inspectors/Profiles/MixedRealityPointerProfileInspector.cs

```diff
@@ -82,7 +82,14 @@ public override void OnInspectorGUI()
                     EditorGUILayout.PropertyField(gazeCursorPrefab, GazeCursorPrefabContent);
                     EditorGUILayout.PropertyField(gazeProviderType);
                     EditorGUILayout.PropertyField(useHeadGazeOverride);
-                    EditorGUILayout.PropertyField(isEyeTrackingEnabled);
+
+                    EditorGUILayout.BeginHorizontal();
+                    EditorGUILayout.PropertyField(useEyeTrackingDataWhenAvailable, new GUIContent("Use Eye Tracking Data"));
```

Comment:
Could the "Use Eye Tracking Data" content be moved to the top of the class with the other `GUIContent`s?

Reply:
Done